app/views/trainees/deferrals/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render PageTitle::View.new(title: "trainees.deferral.show", has_errors: @deferral.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
+    href: edit_trainee_path(@trainee)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_for(@deferral, url: trainee_deferral_path(@trainee), local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <span class="govuk-caption-l">
+        <%= trainee_name(@trainee) %>
+      </span>
+
+      <%= f.govuk_radio_buttons_fieldset(:defer_date_string, legend: { text: "When did the trainee defer?", tag: "h1", size: "l" }, classes: "outcome-date") do %>
+        <%= f.govuk_radio_button :defer_date_string, :today, label: { text: "Today" }, link_errors: true %>
```

Comment:
John also added common translations for "Today", "Yesterday", "Specific date" and "On what date". Please use them here.

Bonus points for also updating the outcome date form whilst you're there!

Reply:
Thanks. Have updated this, will address outcome date in a separate PR